### DISCUSSION TITLE
Бафф карго

### DISCUSSION THREAD

**Discussion by homexp13** [Cosmic-Overlord/TerraGov-Marine-Corps#301] [Ideas]
Значит в чём состоит идея, добавить во вкладке Awaiting Delivery, кнопку быстрой доставки что то вроде такого:
![image](https://github.com/Cosmic-Overlord/TerraGov-Marine-Corps/assets/52973135/ec2366d3-6248-43e7-97ed-272d70330a7b)
Благодаря ней РО сможет непосредственно отправить на маяк заказ избегая лифта
![image](https://github.com/Cosmic-Overlord/TerraGov-Marine-Corps/assets/52973135/73463eda-a9e3-43ae-94ba-c7057b5c98b9)
К тому же при отсутствии РО карго сможет заниматься ИИ, либо синт на планете может принимать заказы с помощью планшета и принимать их

**Comment by fe-ed** [comment]
Может сделать быструю доставку за денежки?
Яндекс еда, все дела

**Reply by homexp13** [reply]
Примерно так и планировал, хотя изначально хотел чтобы у маринов на планете был какой нибудь маяк, который нужно было бы починить и он отвечал за работоспособность данной функции, но это уже будет труднее сделать.

**Reply by fe-ed** [reply]
Да, лучше пока просто денежки

**Reply by Blundir** [reply]
я для вас шутка видимо
